LFS171x/sawtooth-material/sawtooth-tuna/client/index.html

```diff
@@ -1,4 +1,7 @@
 <!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Parts of this code was contributed by Zac Delventhal @delventhalz 
```

Comment:
This code was written by Zac Delventhal @delventhalz.

Reply:
rephrased